### ISSUE TITLE
Icons are broken on example page

### ISSUE THREAD

**Issue by 4eveRS** [andrewgioia/Mana#49]
As the title says, the icons are broken on the example page.

https://andrewgioia.github.io/Mana/icons.html

**Comment by brunoliveiralcantara** [comment]
Yes, the CSS file response 404 from https://andrewgioia.github.io/Mana/node_modules/mana-font/css/mana.min.css in this page as the @4eveRS said.
